source/lambda/customResourceEdgeLambdaCreator/lambda_function.py

```diff
@@ -54,16 +54,24 @@ def on_create(event, context):
         deleted_etag = cf_config.pop('ETag', None)
 
         cf_config['DistributionConfig']['DefaultCacheBehavior']['LambdaFunctionAssociations'] = {
+            'Quantity': 1,
             'Items': [
                 {
-                    'LambdaFunctionARN': edge_lambda_function_arn,
+                    'LambdaFunctionARN': f"{edge_lambda_function_arn}:1",
                     'EventType': 'origin-response',
                     'IncludeBody': False
                 }
             ]
         }
 
+        cf_response = cloudfront_client.update_distribution(
+            DistributionConfig=cf_config['DistributionConfig'],
+            Id=os.environ['CLOUDFRONT_DIST_ID'],
+            IfMatch=deleted_etag)
+        print(f"Updated cloudfront distribution with lambda: {cf_response}")
+
     except Exception as e:
+        print(e)
         raise e
     print("Created edge lambda function with ARN {}".format(
```

Comment:
Shouldnt this print statment come above the newly added changes. It seems to fir better at https://github.com/awslabs/document-understanding-solution/pull/131/files/8ae1dea395b22224774aa877f92f44cdb4f2c2c9..823261b127f2652bd1f9b22851f2358a0ba8a673#diff-fa9025736442f7f15b6b5f18c523569986e7c289cf7074df1f19b456659b61b4R49